website/docs/registry/policy-libraries/publishing.mdx

```diff
@@ -0,0 +1,80 @@
+---
+page_title: Terraform Registry - Publishing Policy Libraries
+description: >-
+  How to publish Policy Libraries to the Terraform Registry.
+---
+
+# Publishing Policy Libraries
+
+A policy library is a collection of [Sentinel policies](https://docs.hashicorp.com/sentinel/writing) and [policy modules](https://docs.hashicorp.com/sentinel/extending/modules) that anyone can share through the Terraform Registry.
+
+Policy libraries are built and maintained on GitHub. Once a policy library has been published, you can release a new version of the policies by pushing a properly formatted Git tag, as described below.
+
+## Requirements for a Policy Library
+
+- **Public GitHub repository**: The files defining Sentinel policies and policy modules must be stored in a public GitHub repository.
+
+- **Repository name**: GitHub repositories containing Policy Libraries must be named according to the convention `policy-library-<NAME>`, where the `<NAME>` segment provides a name for your policy library.
+
+- **Repository description**: The repository's description is displayed in search and browse pages of the Registry. This should be a short, one sentence description of purpose of the policy library.
+
+- **sentinel.hcl file**: The GitHub repository must include a `sentinel.hcl` file at its root level. This file outlines the names and file locations for policies and policy modules that are included in the policy library. Any policy files in the repository that are not listed in this file will be ignored.
+
+    This is a [Sentinel configuration file](https://docs.hashicorp.com/sentinel/configuration), and it uses the same syntax that a Sentinel author is already familiar with. There are two caveats:
+
+    1. [Remote sources](https://docs.hashicorp.com/sentinel/configuration/remote-sources) for policies and policy modules are not allowed.
+    2. The `enforcement_level` for a policy does not need to be included and is ignored by the Registry.
```

Comment:
> It could be nice to have the tiniest little file example here for folks to follow.

Do you think that links to the Sentinel syntax description here is not enough? An example snippet is _very short_ and almost entirely dependent on the context of the library, which gives me some pause. 

Here the `"policy-name"` and path are both at the author's discretion:

```hcl
policy "policy-name" {
  source = "./path/to/somewhere/in/the/repo/policy.sentinel"
}
```